### ISSUE TITLE
Investigate 3rd party tooling options

### ISSUE THREAD

**Issue by mmmavis** [mozilla/schedule-app-core#220]
### Notice: ticket description has been updated! ###

Let's discuss in this Etherpad first: https://public.etherpad-mozilla.org/p/third-party-event-app-checklist. I'll copy things over and update the description of this ticket when the checklist is settled.

**Comment by ioana-chiorean** [comment]
* Lanyrd (http://lanyrd.com/) 
Quick feedback for it - on my side at least for several months now I get **temporary read only website.**
We used it for one MozFest I think - was ok but if we are to be use it again I would test intensive their servers

**Comment by Mte90** [comment]
Sched have the worst integration for the mobile, no notification, you have to add manually the calendar ics url that don't have the content updated.
I think we need to analyze what we need for the event list:
* Notification before the event itself?
* Mark the event do you want follow?
* Get only a schedule?
* Know the people that marked that event?

**Comment by cadecairos** [comment]
http://alternativeto.net/software/sched/
http://alternativeto.net/software/lanyrd/

---------------------------------------------------------
[Meetup](http://www.meetup.com/)
[EventNut](http://go.eventnut.com/)
[EventZilla](http://www.eventzilla.net/) <- I have a strange Bias towards this one, not sure why. ;)

**Comment by mmmavis** [comment]
Sorry for being back and forth on this guys 😬  - let's discuss in this Etherpad first: https://public.etherpad-mozilla.org/p/third-party-event-app-checklist. I'll copy things over and update the description of this ticket when we are settled on the checklist.

**Comment by xmatthewx** [comment]
@edrushka – Before we dive into extending our event app for Mozfest, we're taking stock of thirdparty options. We're drafting a list of [requirements](https://public.etherpad-mozilla.org/p/third-party-event-app-checklist.) and would love your input.

**Comment by edrushka** [comment]
I added a couple notes to the etherpad.

Here's the ticket when we [investigated this last year] (https://github.com/MozillaFoundation/plan/issues/442#issuecomment-101320526) (sadly all our notes were lost in the etherpad apocalypse 😢).

And don't forget to check the [mega bible] (http://books.eventmanagerblog.com/the-event-app-bible/page/1) (careful! you can get lost in here for days!)

**Comment by xmatthewx** [comment]
I filed a ticket to recover the pad https://github.com/MozillaFoundation/mofo-devops/issues/379

**Comment by xmatthewx** [comment]
2015 options reviewed here: http://etherpad.io/p/2015scheltool-recovered

**Comment by xmatthewx** [comment]
I discussed the [2015 review](http://etherpad.io/p/2015scheltool-recovered) of apps with @edrushka, and checked in on our [2016 review](https://public.etherpad-mozilla.org/p/third-party-event-app-checklist). 

There are several blockers that make 3rd party options less desirable and more costly than continued use of our current app given it's current effectiveness.

cc @hannahkane @simonwex 

### Conclusion

We won't pursue a 3rd party option at this time. 
Note: 
 - We should always keep an eye open for new, better options in the future.
 - We need to evaluate which additions we pursue based on estimates (UI, offline, documentation).

### Primary concerns
 - Many admins not possible 
 - Cost
 - Require user accounts (security)
 - Viability for our range of smaller events (3rd party apps too complex or costly)
 - A decent app will likely reveal unexpected problems once we use it

### Secondary concerns
 - Offline capability (we don't currently offer this, but could implement)
 - Branding of flagship event (not a blocker, but not ideal)
 - Can't add awesomeness like documentation

**Comment by xmatthewx** [comment]
R?
 - [ ] @simonwex 
 - [ ] @hannahkane 

Skip the thread. Read [my comment](https://github.com/mozilla/schedule-app-core/issues/220#issuecomment-237386761) just above.

**Comment by hannahkane** [comment]
I'm surprised by this conclusion. Many organizations run events using 3rd party apps, and I'm having trouble understanding why our needs are particularly different. 

Are there no 3rd party apps that allow for multiple admins? Does the cost of using a 3rd party app really outweigh the cost of building and maintaining our own? I don't think the user account requirement should necessarily preclude using an existing app, though we should of course avoid apps with specific security issues. I'd need to understand the issue of smaller events more—I've been to several smallish events (couple hundred people) that made use of a 3rd party tool, and really small events (for example events without multiple tracks) don't seem to need a schedule app at all. The point about unexpected problems arising seems like it could be countered by the popularity or positive reviews of existing apps. 

Bottom line: This feels like a solved problem to me, so I question our need to build something ourselves. Having said all that, I was excited about the possibilities around leveraging an app for connecting people beyond in-person events. That, to me, is the main argument for building our own.

**Comment by xmatthewx** [comment]
Let's discuss over video with @simonwex and @edrushka.

> I question our need to build something ourselves. 

The thing is, we already built it. And it requires very little maintenance. We could clean up the header and use it more or less as is. 

> I was excited about the possibilities around leveraging an app for connecting people beyond in-person events. That, to me, is the main argument for building our own.

I am (and I believe we are) also excited about building out the app to help connect people beyond the events. Sticking with the app today keeps that option open whether we develop some or all of that feature set now or later. 

> Are there no 3rd party apps that allow for multiple admins?

A high priced app might enable the features we already have. Is that enough to justify the cost? (Particularly if that cost makes it inaccessible for other hub/network events?)

> The point about unexpected problems arising seems like it could be countered by the popularity or positive reviews of existing apps.

Mozfest is coming soon. There is risk in trying a new app. Is there a clear benefit?

> I don't think the user account requirement should necessarily preclude using an existing app

If the app collects personal info, I believe it needs to run through security review. 

> understand the issue of smaller events more

I think this is more about the opportunities. Still TBD. If we can make our existing tech easy to spin up for other events in our network, we can benefit those events. 

Lastly, I'd be glad to test an app if it fits our criteria. I know our search hasn't been exhaustive, but no clear option has jumped out as better than what we have already.

**Comment by edrushka** [comment]
Have we confirmed we'll use the MoFo Schedule App for MozFest or is Sched/a 3rd party tool still in discussion?

**Comment by xmatthewx** [comment]
Yes, we are confirmed. We will use our existing tool (with a few easy to achieve improvements) for Mozfest this year. After, we will reopen the search for 3rd party alternatives.
